src/meta/model_v2/src/lib.rs

```diff
@@ -192,38 +282,57 @@ impl From<BTreeMap<u32, Vec<u32>>> for ActorUpstreamActors {
     }
 }
 
-derive_from_json_struct!(DataType, risingwave_pb::data::DataType);
-derive_from_json_struct!(DataTypeArray, Vec<risingwave_pb::data::DataType>);
-derive_from_json_struct!(FieldArray, Vec<risingwave_pb::plan_common::Field>);
+derive_from_byte_array!(StreamNode, PbStreamNode);
```

Comment:
May add some simple comments on whether `derive_from_byte_array` or `derive_from_json_struct` should be used.